src/lib/openjp2/opj_includes.h

```diff
@@ -44,6 +44,10 @@
    Standard includes used by the library
  ==========================================================
 */
+
+/* Get declarations of fseeko, ftello, posix_memalign. */
+#define _POSIX_C_SOURCE 200112L
```

Comment:
opj_includes.h gets installed. So we are polluting everyone else code with our setting. Can't this be moved to say opj_config_private.h.cmake.in ?